src/theme/ItaliaTheme/Subsites/ItaliaTheme/_common.scss

```diff
@@ -77,7 +77,11 @@
   //btn mini
   .select-all-cts.btn.btn-link.btn-mini,
   .subsite-header .text a.select-all-cts.btn-link.btn-mini {
-    color: $subsite-primary !important;
+    @if $subsite-light-theme {
+      color: $subsite-secondary !important;
```

Comment:
Ho un dubbio su questo: non abbiamo la garanzia che sia un colore scuro il secondario se il tema è chiaro, o no? Forse meglio `$subsite-primary-text`? Ma non sono nemmeno sicuro di aver capito bene che bottoni siano questi, quindi forse il commento è inutile. Riesci a darmi un indizio in più?

Reply:
Sono questi che sembrano dei link, esempio nel filtro di ricerca

<img width="1495" alt="Screenshot 2024-03-04 alle 12 42 36" src="https://github.com/RedTurtle/design-comuni-plone-theme/assets/43245702/7b3146c2-baf0-4191-a4c0-112eb829d04a">

Se dici che il secondario potrebbe non essere il colore accentuato con cosa potrei cambiarlo, con quello dei link?

Reply:
@SaraBianchi ci sta di metterci il colore dei link secondo me, dato che sono proprio bottoni che sembrano link. Mi sembra la scelta più coerente.

Reply:
@pnicolli perfetto, aggiornato!